packages/web3-eth-contract/src/types.ts

```diff
@@ -75,76 +51,6 @@ export interface ContractEventOptions {
 	topics?: string[];
 }
 
-export interface ContractOptions {
```

Comment:
for all types moved from contracts to other packages, reexport these from contracts so its not breaking.